ember/app/caluma-query/models/work-item.js

```diff
@@ -0,0 +1,56 @@
+import WorkItemModel from "@projectcaluma/ember-core/caluma-query/models/work-item";
+
+export default class CustomWorkItemModel extends WorkItemModel {
+  get isReady() {
+    return this.raw.status === "READY";
+  }
+
+  get isCompleted() {
+    return this.raw.status === "COMPLETED";
+  }
+
+  get case() {
+    return this.raw.case.parentWorkItem?.case || this.raw.case;
```

Comment:
I don't see where we're fetching the `parentWorkItem`

Reply:
Removed